test/unit/presenters/formats/completed_transaction_presenter_test.rb

```diff
@@ -75,7 +75,6 @@ def result
     edition.update_attribute(:slug, 'foo')
     expected = [
       { path: '/foo', type: 'prefix' },
```

Comment:
Honestly I just missed deleting the comma, but I can them in to the other examples if required.